libkbfs/kbfs_ops_test.go

```diff
@@ -337,14 +337,15 @@ func TestKBFSOpsGetRootNodeReIdentify(t *testing.T) {
 	require.NoError(t, err)
 	assert.True(t, fboIdentityDone(ops))
 
-	// The channel is not buffered, when the second value is received
-	// the first round of marking for reidentify will be done.
+	// Mark everything for reidentifying, and wait for it to finish
+	// before checking.
 	kop := config.KBFSOps().(*KBFSOpsStandard)
-	kop.reIdentifyControlChan <- struct{}{}
-	kop.reIdentifyControlChan <- struct{}{}
+	returnCh := make(chan struct{})
+	kop.reIdentifyControlChan <- returnCh
+	<-returnCh
```

Comment:
Yes, the control channel is only for tests, and this is the only test that uses it.